includes/class-pureclarity-order.php

```diff
@@ -40,7 +40,7 @@ public function get_order_info( $order_id ) {
 						if ( is_object( $product ) ) {
 							$order_info['items'][] = array(
 								'id'        => $item->get_product_id(),
-								'qty'       => $item['qty'],
+								'qty'       => ceil( $item['qty'] ),
```

Comment:
just wondering what happens if we dont have qty?

before would have set property to undefined... now will throw error?